### DISCUSSION TITLE
Church 🔔🔔 are mainly made of what metal?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1139] [Galaxy Trivia] [answered]
**Bonus:** Which way do you go when you go widdershins?

**Comment by peterbaccaro** [comment]
Church bells are made from **bronze** - an alloy consisting of 78 percent copper and 22 percent tin. 

Widdershins is a term meaning to go **counter-clockwise, anti-clockwise, or left-hand-wise, or to walk around an object by always keeping it on the left**.

**Comment by thuongtruong1009** [comment, marked as answer]
Church bells are typically made of bronze or a brass alloy, as these metals produce a resonant and melodious sound when struck.

When you go widdershins, you are moving in a counterclockwise or anticlockwise direction. It is often associated with going against the natural or traditional order, and it can be considered a direction of superstition or bad luck in some cultures and belief systems.
